primitives/src/sybil.rs

```diff
@@ -164,10 +164,10 @@ impl PersonhoodUniquenessRating {
 }
 
 pub fn sibling_junction(id: u32) -> (Junction, Junction) {
-    (Junction::Parent, Junction::Parachain(id))
+    (Junction::OnlyChild, Junction::Parachain(id))
```

Comment:
@clangenb can you please check if this is legit? it makes the build work and the test pass, but not sure about the meaning TBH

Reply:
hmm, actually, test fails

Reply:
hmm, I see that they have changed quite some things, but this seems wrong. I have only checked briefly. I think we should probably use this:

```
MultiLocation {
   parents: 1,
   interior: X1(Parachain(id),
}
```

Reply:
I'm actually not sure we need to switch to v1 at all, but I just couldn't make v0 build anymore

Reply:
works with v1 now. you were spot on